.github/workflows/build.yml

```
@@ -130,13 +136,16 @@ jobs:
           toolchain: "1.71"
 
       # zlib-rs is optional and requires a later MSRV
-      - run: cargo check --locked --lib $(admin/all-features-except zlib ${{ env.MAIN_CRATE }}) -p ${{ env.MAIN_CRATE }}
+      - run: cargo check --locked --lib ${{ env.ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB }} -p ${{ env.MAIN_CRATE }}
+        env:
+          # (also need to exclude `unstable-api-all` which includes `zlib`)
+          ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB: $(admin/all-features-except unsafe-assume-single-core,unstable-api-all,zlib ${{ env.MAIN_CRATE }})
```

minor nit, likely not worth fixing at this point: I think longer admin operations like this should be done by another admin script, for example:
```suggestion
          ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB: $(admin/all-crate-features-except-unsafe-and zlib ${{ env.MAIN_CRATE }})
```

This could likely benefit proposed updates to `admin/coverage` script as well.